src/constants.ts

```diff
@@ -33,3 +33,7 @@ export const CONST_URL_MONITOR_GITHUB =
 export const CONST_DAY_MILLISECONDS = 24 * 60 * 60 * 1000;
 export const CONST_URL_MONITOR_LANDING_PAGE_ID =
   "monitor.mozilla.org-monitor-product-page";
+export const CONST_MOZILLA_ACCOUNTS_SETTINGS_PROMO_SEARCH_PARAMS = {
```

Comment:
Since we currently only expect FxA to want to initiate this, should we also check for the `utm_source`?

Reply:
Added in [de361e6](https://github.com/mozilla/blurts-server/pull/5104/commits/de361e69886cb09de503d3e032e638c638e1d89b).